extern/storage-sealing/commit_batch.go

```diff
@@ -192,12 +270,14 @@ func (b *CommitBatcher) processBatch(notif, after bool) (*cid.Cid, error) {
 }
 
 // register commit, wait for batch message, return message CID
-func (b *CommitBatcher) AddCommit(ctx context.Context, s abi.SectorNumber, in AggregateInput) (mcid cid.Cid, err error) {
+func (b *CommitBatcher) AddCommit(ctx context.Context, s SectorInfo, in AggregateInput) (mcid cid.Cid, err error) {
+	sn := s.SectorNumber
 	b.lk.Lock()
-	b.todo[s] = in
+	b.sectors[sn] = s
```

Comment:
As mentioned above, maybe store sector deadline instead of sector info (the deadline shouldn't change)

Reply:
yeah, good point!